security/email-templates.md

```diff
@@ -167,3 +167,42 @@ Thanks,
 
 $PERSON (on behalf of the Envoy security team and maintainers)
 ```
+
+## Security Fix of Main Branch Announcement
+
+```
+Subject: Security fix of Envoy main branch (that includes $GITSHORTCOMMITHASH) is now available
+To: envoy-security-announce@googlegroups.com
+Cc: envoy-announce@googlegroups.com, envoy-security@googlegroups.com, envoy-maintainers@googlegroups.com
+
+Hello Envoy Community,
+
+The Envoy security team would like to announce the availability of the security fix for security
+defects introduced in the main branch by [$GITSHORTCOMMITHASH]($GITHUBCOMMITURL) commit. The defects
+caused by [$GITSHORTCOMMITHASH]($GITHUBCOMMITURL) is not part of any Envoy stable releases.
+
+$DEFECTSSUMMARY
+
+<!-- Provide details on features, extensions, configuration that make it likely that a system is
+vulnerable in practice. -->
+
+The CVSS score for this is [$CVSSSTRING]($CVSSURL).
+
+Including this [$FIXGITSHORTCOMMITHASH]($FIXGITHUBCOMMITURL) commit is encouraged to fix this issue.
```

Comment:
s/this/the/

Reply:
Change CVSS to CVSSv2 or CVSSv3 depending which version you choose.

Reply:
Hum, I need to defer to @envoyproxy/security-team for choosing the right version. I can do the find-and-replace, once it is decided.

Reply:
We use CVSSv3, but the actual CVSS string and URL contain this information, so I don't feel it's necessary to call that out.